scripts/remote_replace_cloudbeat.sh

```diff
@@ -1,22 +1,46 @@
 #!/bin/bash
 
+set -eo pipefail
+
 # This script uses the kubectl commands in order to ssh into the cluster defined in the host current-context
 # The script lets you to remotely replace the cloudbeat binary with a file named cloudbeat, located on the host's running directory
 
 source ./scripts/common.sh
 
-LOCAL_BINARY="./cloudbeat"
+ARCH=$(find_target_arch)
+if [ "$ARCH" = "amd64" ]; then
+    ARCH="x86_64"
+fi
+OS=$(find_target_os)
+VERSION=$(make get-version)
+
+LOCAL_DIR=cloudbeat-$VERSION-SNAPSHOT-$OS-$ARCH
+echo "Looking for build distribution: $LOCAL_DIR"
+tar -xvf build/distributions/$LOCAL_DIR.tar.gz > /dev/null 2>&1
 
-PODS=$(kubectl -n kube-system get pod -l app=elastic-agent -o name)
-for P in $PODS; do
+for P in $(get_agents); do
   POD=$(echo $P | cut -d '/' -f 2)
-  BINARY_FILEPATH="$(find_cloudbeat_binary $POD)"
-  if [ -z "$BINARY_FILEPATH" ]
+  SHA=$(get_agent_sha $POD)
+  echo "Found sha=$SHA in pod=$POD"
+  
+  ROOT=/usr/share/elastic-agent/data/elastic-agent-$SHA
+  DEST=$ROOT/components
+  cp_to_pod $POD $LOCAL_DIR/cloudbeat $DEST
+  cp_to_pod $POD $LOCAL_DIR/cloudbeat.yml $DEST/cloudbeat.yml
```

Comment:
not sure we'll always want to copy `cloudbeat.yml`, wdyt?

Reply:
Right, this is not the behavior we have today.